packages/website/ts/types.ts

```diff
@@ -15,6 +15,7 @@ export interface Token {
     decimals: number;
     isTracked: boolean;
     isRegistered: boolean;
+    trackedTimestamp?: number;
```

Comment:
Miiiiiiiight want to consider consolidating `isTracked` and `trackedTimestamp` just because having both opens us up to bugs where they are inconsistent (ex: isTracked  = true, trackedTimestamp: undefined }. So this would be mean just having `trackedTimeStamp?`. Also blockchain.ts does some work with `isTracked` -- is that still valid?

Reply:
My only worry about dropping the isTracked field is for users who already have tracked tokens in storage, we may be dropping them

Reply:
Oh good point. The backwards compat argument is good. How will those users experience adding new tokens? Will we be prepending to Wallet or appending? If it's pre-pending, then it might be worth writing a little migration script anyway.

Reply:
We do have some utilities for clearing tracked tokens local storage for backwards incompatible changes

Reply:
Up to you! I do think the chances of us running into this inconsistency in an unexpected way are relatively high down the line. Maybe this falls under https://app.asana.com/0/search/720658791610697/690670364591515

Where we would redux-ify all this logic and call moment().unix() in the reducer (and maybeee get rid of isTracked)